src/PackageDetailsViewExtension/PackageDetailsView.xaml.cs

```diff
@@ -8,6 +9,11 @@ namespace Dynamo.PackageDetails
 {
     public partial class PackageDetailsView : UserControl
     {
+        /// <summary>
+        ///     Used to notify when this control is closed 
+        /// </summary>
+        public event EventHandler Closed;
```

Comment:
`internal`?

Reply:
I was using this to subscribe from inside the Package Details ViewExtension, but I guess we have already made the internal properties visible to this extension, so making this internal was totally fine!